pkg/gardenlet/controller/shoot/care/health.go

```diff
@@ -447,6 +468,13 @@ func (h *Health) checkObservabilityComponents(
 			return exitCondition, err
 		}
 	}
+
+	if exitCondition := h.checkManagedResources(condition, managedResources, func(managedResource resourcesv1alpha1.ManagedResource) bool {
+		return sets.New(string(gardencorev1beta1.ShootObservabilityComponentsHealthy)).Has(managedResource.Labels[v1beta1constants.LabelCareConditionType])
```

Comment:
I guess it is more similar to the other filter functions, but would the following not be the same (and more concise)?

```
return managedResource.Labels[v1beta1constants.LabelCareConditionType] == string(gardencorev1beta1.ShootObservabilityComponentsHealthy)
```

Reply:
😄 Yes, indeed, this is changed in a later commit :) You will see it when you proceed with the review.